pkgs/applications/misc/mupdf/default.nix

```diff
@@ -166,11 +170,7 @@ stdenv.mkDerivation rec {
     EOF
 
     moveToOutput "bin" "$bin"
-  '' + (lib.optionalString (stdenv.isDarwin) ''
-    for exe in $bin/bin/*; do
-      install_name_tool -change build/shared-release/libmupdf.dylib $out/lib/libmupdf.dylib "$exe"
-    done
-  '') + (lib.optionalString (enableX11 || enableGL) ''
+  '' + (lib.optionalString (enableX11 || enableGL) ''
```

Comment:
POC of my PR. This is how you can use it, this is what people won't have to do manually anymore.